src/v3/src/components/ImageWithText/ImageWithText.tsx

```diff
@@ -23,21 +23,27 @@ const ImageWithText: UISchemaElementComponent<{
   uischema: ImageWithTextElement
 }> = ({ uischema }) => {
   const Icon = uischema.options.SVGIcon;
+  const { textContent } = uischema.options;
 
   return (
     <Box
       id={uischema.options.id}
       display="flex"
-      justifyContent="flex-start"
+      justifyContent={textContent ? 'flex-start' : 'center'}
```

Comment:
image was not being centered properly

Reply:
Pass `uischema.options.alignment` to `ImageWithText` (default: `flex-start`) for now. Eventually hoping to convert to a markdown renderer